governance/votes/Draft- Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Template - [Executive Vote] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Delegate Compensation, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank and Other Actions - September 13, 2023
+summary: Scope-defined parameter changes; Spark Protocol D3M parameter changes; Aligned Delegate compensation for August 2023; set Fortunafi Debt Ceiling to zero; approve DAO Resolution for HV Bank to terminate future purchases; Trigger Spark Protocol Proxy Spell; Core Unit MKR vesting transfers; scuttle MCD_CAT contract.  
+date: 2023-09-13T00:00:00.000Z
+address: "$spell_address"
+
+
+
+
+
+---
+
+# [Executive Proposal] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Delegate Compensation, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank and Other Actions - September 13, 2023
+
+The Reserve Governance Facilitators, Sidestream, Dewiz and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
```

Comment:
I believe this needs to be restructured. While it can be occasionally confusing, GovAlpha's approach of categorizing these changes and new additions seems somewhat beneficial for clarity. Furthermore, they should be ordered from top to bottom in terms of significance. I'm open to deferring to LB's judgment on this matter, but my suggested order is more as follows:

- The Fortunafi vault's Debt Ceiling will be set to zero DAI, per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
- The `MCD_CAT` contract will be scuttled by removing it from the Chainlog and removing all its permissions on the Protocol contracts.
- A Spark Protocol Proxy Spell will be triggered that implements changes to the DAI borrow spread and the flash loan fee.
- Several Stability Scope-defined parameter changes will be executed, as detailed below.
- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased to 400 million DAI; and the Ceiling Increase Cooldown will be increased to 12 hours. 
- Aligned Delegate compensation for August 2023 will be distributed.
- MKR vesting transfers will be executed for the DECO-001 and SES-001 Core Units, per their successful budget proposals.

Reply:
Thanks JS. 

I'll give things another round of consideration, but I think some of this is probably fluid/subjective. 

For instance, the MCD_CAT item is just housekeeping; the contract hasn't been in use for 3 years. I don't think this is a high-impact/importance item that belongs near the top of the Exec Copy.

The Spark Protocol Proxy Spells are typically listed near-ish the bottom of previous Exec Votes. And it seems these are relatively non-major changes.